src/config.ts

```diff
@@ -79,6 +79,7 @@ const {
   STATSD_PORT,
   TINEYE_API_PASSWORD,
   TINEYE_API_USERNAME,
+  TINEYE_ENVIRONMENT,
```

Comment:
Wondering if we should give this env variable a more generic name to make it reusable for similar cases. Something like `SYSTEM_ENVIRONMENT` could make sense.

Reply:
👍🏽